site/docs/v1/tech/core-concepts/search.adoc

```diff
@@ -81,4 +82,29 @@ include::docs/v1/tech/shared/_re-indexing-elasticsearch.adoc[]
 
 include::docs/v1/tech/shared/_paginating-search-results.adoc[]
 
-Note that you'll only be able to get back 10000 results no matter how you paginate. See the link:/docs/v1/tech/reference/limitations#user-searches[Known Limitations] section for more and some workarounds.
+Note that prior to version 1.48.0 you'll only be able to get back 10000 results no matter how you paginate. See the link:/docs/v1/tech/reference/limitations#user-searches[Known Limitations] section for more and some workarounds.
+
+[NOTE.since]
+====
+
+As of version 1.48.0, FusionAuth you can use a [field]#nextResults# token to page past the 10000 results limitation when using the Elasticsearch engine.
+
+See the <<Extended Pagination>> section for more details.
+
+====
+
+=== Extended Pagination
```

Comment:
I think there should be a `since` note at the beginning of this section that just says "Available since 1.48.0" Maybe change the one in the previous section to just be regular text?